src/components/home/PlatformsList.jsx

```diff
@@ -0,0 +1,17 @@
+import { platformsLogos } from './platformsLogos';
+
+export default function PlatformsList({ game, isLoaded }) {
+  return (
+    <>
+      <ul className="flex items-center gap-1">
+        {isLoaded ? (
+          game.parent_platforms.map((platform, platformIndex) => (
+            <li key={platformIndex}>{platformsLogos(platform.platform)}</li>
+          ))
```

Comment:
Petit tip, tu peux déconstruire ici
```suggestion
          game.parent_platforms.map(({ platform }, platformIndex) => (
            <li key={platformIndex}>{platformsLogos(platform)}</li>
          ))
```